common.yml

```diff
@@ -753,19 +756,68 @@ functions:
               --package-file-suffix "" \
               release.msi
 
+  "sign macos zip":
+    - command: shell.exec
+      params:
+        working_dir: src/github.com/mongodb/mongo-tools/
+        silent: false
+        script: |
+          set -o xtrace
+          set -o errexit
+          set -o verbose
+          if [ "${mongo_os}" != "osx" ]; then
+              exit 0
+          fi
+          curl -sL https://github.com/mitchellh/gon/releases/download/v0.2.3/gon_macos.zip > gon.zip
+          unzip gon.zip
+          OUTPUT_PATH="./release.zip"
+          AC_USERNAME="${ac_username}"
+          AC_PASSWORD="${ac_password}"
+          # gon settings
+          cat <<EOF_GON_JSON > gon.json
+          {
+            "source" : ["LICENSE.md", "README.md", "THIRD-PARTY-NOTICES",
+              "bin/bsondump", "bin/mongodump",  "bin/mongoexport",
+              "bin/mongofiles", "bin/mongoimport",  "bin/mongorestore",
+              "bin/mongostat",  "bin/mongotop"
+            ],
+            "bundle_id" : "com.mongodb.mongotools",
+            "apple_id": {
+              "username": "$AC_USERNAME",
+              "password": "$AC_PASSWORD"
+            },
+            "sign" :{
+              "application_identity" : "Developer ID Application: MongoDB, Inc. (4XWMY46275)"
+            },
+            "zip" :{
+              "output_path": "$OUTPUT_PATH"
+            }
+          }
+          EOF_GON_JSON
+          tar xvzf release.tgz
+          mv mongodb-database-tools*/* ./
+          ssh -v -p 2222 localhost -t "cd $PWD && ./gon -log-level=info gon.json"
+
   "sign linux packages":
     - command: shell.exec
       type: system
       params:
         working_dir: src/github.com/mongodb/mongo-tools/
         silent: true
         script: |
+          if [ "${mongo_os}" = "osx" ]; then
+              exit 0
+          fi
           echo "${signing_auth_token_server_4_4}" > ./signing_auth_token
     - command: shell.exec
       type: system
       params:
         working_dir: src/github.com/mongodb/mongo-tools/
+        silent: false
```

Comment:
This line will help clarify to anyone coming around why this script is split in two parts.